examples/ex1.cpp

```diff
@@ -75,7 +75,7 @@ int main(int argc, char *argv[])
                   "--no-static-condensation", "Enable static condensation.");
    args.AddOption(&pa, "-pa", "--partial-assembly", "-no-pa",
                   "--no-partial-assembly", "Enable Partial Assembly.");
-   args.AddOption(&device, "-d", "--device",
+   args.AddOption(&device_opts, "-d", "--device",
```

Comment:
Can we rename `device_opts` to `device_config`? We call it the device "configuration string"...

Reply:
Sounds good to me.